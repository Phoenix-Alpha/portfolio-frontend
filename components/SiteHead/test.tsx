import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { SiteHead } from './index'

describe('<SiteHead />', () => {
  it('should renders correct <SiteHead />', async () => {
    const wrapper = shallow(<SiteHead />)

    await new Promise((resolve) => setTimeout(resolve, 0))

    wrapper.update()

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
