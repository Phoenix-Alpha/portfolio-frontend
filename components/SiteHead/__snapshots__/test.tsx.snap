// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SiteHead /> should renders correct <SiteHead /> 1`] = `
<Head>
  <title>
    Flatrock - Invest in private companies
  </title>
  <meta
    charSet="utf-8"
  />
  <meta
    content="Invest in private companies"
    name="description"
  />
  <meta
    property="og:url"
  />
  <meta
    content="website"
    property="og:type"
  />
  <meta
    content="Flatrock - Invest in private companies"
    property="og:title"
  />
  <meta
    content="Invest in private companies"
    property="og:description"
  />
  <link
    rel="preconnect"
  />
</Head>
`;
